test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -17,25 +18,30 @@ define([
         fakeSpecs = {
             someApp:
                 TestAppObj.app.specs['kb_uploadmethods/import_fastq_sra_as_reads_from_staging'],
-        },
-        runtime = Runtime.make();
+        };
 
-    describe('test the bulk import cell module', () => {
+    xdescribe('test the bulk import cell module', () => {
```

Comment:
Skipping these for now.

Other tests happen to cover ~1/2 of the module, though.